lib/phoenix/util.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Phoenix.Util do
```

Comment:
We don't need a separate module for this, can you please define it direclty in phx.gen.context? Tks.

Reply:
done